ui/js/models/perf/alerts.js

```diff
@@ -217,6 +217,23 @@ treeherder.factory('PhAlerts', [
             return _.find(phAlertSummaryStatusMap, { id: this.status }).text;
         };
 
+        AlertSummary.prototype.notesChanged = (this.notes !== this.original_notes);
+        AlertSummary.prototype.saveNotes = function () {
+            const alertSummary = this;
+            return $http.put(getApiUrl(`/performance/alertsummary/${this.id}/`),
```

Comment:
I think you could use `alertSummary.modify({notes: this.notes })` instead of repeating the API call here.